.github/workflows/backport.yml

```diff
@@ -27,3 +27,43 @@ jobs:
           BACKPORT_LABEL_REGEXP: "backport/(?P<target>\\d+\\.\\d+\\.[+\\w]+)"
           BACKPORT_TARGET_TEMPLATE: "release/{{.target}}"
           GITHUB_TOKEN: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
+  handle-failure:
+    needs:
+      - backport
+    if: always() && needs.backport.result == 'failure'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send slack notification on failure
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          payload: |
+            {
+              "text": ":x::arrow_right_hook::nomad-sob: Backport run *FAILED*",
+              "attachments": [
+                {
+                  "color": "#C41E3A",
+                  "blocks": [
+                    {
+                      "type": "section",
+                      "fields": [
+                        {
+                          "type": "mrkdwn",
+                          "text": "*Pull Request:*\n<${{ github.event.pull_request.html_url}}|${{ github.repository }}#${{ github.event.pull_request.number}}>"
+                        },
+                        {
+                          "type": "mrkdwn",
+                          "text": "*From:*\n@${{ github.event.sender.login }}"
+                        },
+                        {
+                          "type": "mrkdwn",
+                          "text": "*Run:*\n<${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|${{ github.run_id }}>"
```

Comment:
Is this supposed to be a link? If I look on Slack it shows as:

> </hashicorp/nomad-enterprise/actions/runs/1|1>

Reply:
Ah yeah, those were the tests I did using [`act`](https://github.com/nektos/act), which don't have a GitHub Action metadata. Running from GitHub we get something like this:
<img width="311" alt="image" src="https://user-images.githubusercontent.com/775380/201181648-cd2ade12-765b-4204-8dc5-e96d66514e14.png">